src/test.rs

```diff
@@ -70,32 +70,41 @@ fn is_equal(lhs: &NodeRef, rhs: &NodeRef) -> Result<(), CompareError> {
 
 #[test]
 fn test_spec() {
-    let src_path = Path::new("tests").join("metadata001.bs");
-    let target_path = Path::new("tests").join("metadata001.html");
+    // TODO: Use all files.
+    let names = hashset! {
```

Comment:
Why a `hashset!`, this looks like it can be an array, and if it has more elements then we depend on iteration order which isn't great.